develop/tutorials/articles/260-themes-and-layout-templates/01-themes/01-theme-generator/01-installing-the-generator.markdown

```diff
@@ -0,0 +1,132 @@
+        Windows, make sure to add the appropriate directory to your system path. 
+        For example, on UNIX you'd set this:
+
+                PATH=${PATH}:${NPM_PACKAGES}/bin
+
+3.  Install 
+    [Yeoman](http://yeoman.io/) 
+    and 
+    [gulp](https://www.npmjs.com/package/gulp) 
+    globally:
+
+        npm install -g yo gulp
+
+4.  With its dependencies installed, execute the following command to install 
+    the Liferay Theme Generator globally:
+
+        npm install -g generator-liferay-theme
+
+The Liferay Theme Generator uses Sass, which requires some additional setup on 
+Windows. If you're not on Windows, you can jump to 
+[running the Liferay Theme Generator](#running-the-themes-generator).
+
+## Installing Sass on Windows [](id=installing-sass-on-windows)
+
+To use Sass on Windows, you must use either Sass from node-sass or Sass from
+Ruby. By default, the generator creates theme projects to use node-sass; but you
+can reconfigure them to use Ruby based Sass and Compass. Since node-sass
+indirectly requires Visual Studio, developers who are not already using Visual
+Studio may opt to use Ruby based Sass and Compass instead of node-sass. 
+
++$$$
+
+**Note:** If your theme was built with an older version of the Liferay Theme 
+Generator and specifies the `"liferay-theme-deps-7.0": "1.0.0"` dependency in 
+its `package.json`, `npm install` may fail in Liferay DXP due to its dependency 
+on node-sass `v3.13.1`. To fix this issue, change the dev dependency in your 
+`package.json` to `"liferay-theme-deps-7.0": "*"` and rebuild your theme. We 
+recommend that you use the latest version of the Theme Generator and 
+dependencies in your `package.json` to ensure full compatibility. 
+
+$$$
+
+This section explains both Sass installations. 
+
+### Installing Sass from node-sass [](id=installing-sass-from-node-sass)
+
+By default, the generator uses Sass from node-sass. node-sass requires node-gyp,
+which in turn requires Python and Visual Studio. The 
+[node-gyp installation instructions](https://github.com/nodejs/node-gyp#installation)
+explain how to set up node-gyp, Python, and Visual Studio. Since Visual Studio 
+is a particularly large dependency, if you aren't already using Visual Studio 
+you might want to consider using Ruby Sass instead of node-sass. 
+
+### Installing Ruby Sass and Compass [](id=installing-ruby-sass-and-compass)
+
+As an alternative to using Sass from node-sass, you can use Sass from Ruby. 
+Follow these steps to set it up:
+
+1.  Install Ruby via the 
+    [Ruby installer](http://rubyinstaller.org/).
+    
+2.  @product@ themes require Compass to be used with Ruby based Sass. Install 
+    the Compass and Sass Ruby gems. Note that Sass version 3.5 has compatibility 
+    issues with Compass. We recommend that you install Sass version 3.4.0, as 
+    shown in the command below:
+    
+        gem install sass -v "=3.4.0"
+
+        gem install compass
+        
```

Comment:
Is Sass V 3.4.0 still the recommended version to install?

Reply:
I'm assuming that hasn't changed.